### ISSUE TITLE
Wrong value from API used? New tariff update missed?

### ISSUE THREAD

**Issue by FlexxFR** [barisdemirdelen/homeassistant-greenchoice#17]
Today new tariff value should be applied, however they are not reflected via this integration.
However, it might be an issue where the data is collected  as it's **also wrong on the website!**

Screenshot 01-Oct-2024
![image](https://github.com/user-attachments/assets/299061b7-e675-41b9-b6af-ac30f1d402d4)

Screenshot taken on same day, but other menu: (Tarieven)
![image](https://github.com/user-attachments/assets/1e790a3e-1487-4ce5-83c0-e03fea2dff1e)

Clearly different values.
So, I hope this will resolves itself in a day or two...but I believe this second source could also be used (referred to in other thread), as future price updates are available here also. Maybe this source is more reliable?

**Comment by FlexxFR** [comment]
I believe Greenchoice updated their webpage (this morning it was in maintenance) and the API URL is changed.

**Previous (from previous threads:**
https://mijn.greenchoice.nl/api/v2/Rates/2177418?AgreementIdElectricity=2776016&AgreementIdGas=2776016&HouseNumber=<CORRECT NUMBER>&ReferenceIdElectricity=179687&ReferenceIdGas=&ZipCode=<CORRECT ZIPCODE>

**Current website:**
https://mijn.greenchoice.nl/api/v2/customers/2177418/rates?AgreementIdElectricity=2776016&HouseNumber=<CORRECT NUMBER>&ReferenceIdElectricity=180620&ZipCode=<CORRECT ZIPCODE>

Moreover:  **"terugleverKostenIncBtw": 0.11616,** is now part of the information that can be retrieved. (see other request)

```
{
  "beginDatum": "2024-10-03T00:00:00",
  "eindDatum": "2024-10-03T00:00:00",
  "stroom": {
    "leveringHoog": 0.12227,
    "leveringLaag": 0.13228,
    "leveringEnkel": 0.12719,
    "leveringLaagAllIn": 0.2917068,
    "leveringHoogAllIn": 0.2795947,
    "leveringEnkelAllIn": 0.2855479,
    "leveringHoogBtw": 0.0485247,
    "leveringLaagBtw": 0.0506268,
    "leveringEnkelBtw": 0.0495579,
    "soortMeter": "DT",
    "rebTeruggaveIncBtw": -631.39009999395,
    "terugLeveringEnkel": 0.12719,
    "terugLeveringHoog": 0.12227,
    "terugLeveringLaag": 0.13228,
    "terugleverVergoeding": 0.145,
    "terugleverKostenIncBtw": 0.11616,
    "terugleverKostenExcBtw": 0.096,
    "terugleverKostenBtw": 0.02016,
    "btw": 0.1487094,
    "btwPercentage": 21,
    "vastrechtPerDagExcBtw": 0.253178,
    "vastrechtPerDagIncBtw": 0.30634538,
    "vastrechtPerDagBtw": 0.05316738,
    "netbeheerPerDagExcBtw": 0.9641,
    "netbeheerPerDagIncBtw": 1.166561,
    "netbeheerPerDagBtw": 0.202461,
    "reb": 0.1088,
    "sde": 0,
    "capaciteit": "3x25A"
  },
  "gas": null
}
```

**Comment by barisdemirdelen** [comment]
Should be fixed with #19. We are using this api now.

**Comment by FlexxFR** [comment]
Great fix. Works fine. Thanks again
